app/controllers/application_controller.rb

```diff
@@ -61,17 +60,6 @@ def valid_help_actions
     @valid_help_actions ||= %w[signed_in create].freeze
   end
 
-  def tfa_feature_flag?
-    Rails.configuration.enable_enhanced_2fa_experience
-  end
-
-  def choose_two_factor_method
-    return if current_user.nil? || !current_user.second_factor_method.nil?
-
-    # TODO: Replace with route helper once fully migrated to enhanced 2FA.
-    redirect_to "/users/two_factor_authentication/setup"
-  end
-
   def confirm_two_factor_setup
     return unless current_user &&
       current_user.need_two_factor_authentication?(request) &&
```

Comment:
This will no longer need request passed once the linter issue is fixed.

Reply:
It does need something passed in because it overloads https://github.com/Houdini/two_factor_authentication/blob/7e0faa6e1f3ab497767c59e1e97fb01e340b59ce/lib/two_factor_authentication/models/two_factor_authenticatable.rb#L59